functions/src/storage.ts

```diff
@@ -0,0 +1,49 @@
+import { db } from "./index";
+import { storage} from 'firebase-admin';
+import { writeFileSync } from "fs";
+
+
+export function StorageWriteList(list: string){
+
+    const documentReference = db.collection('list').doc(list).collection('item');
+
+    documentReference.get().then((doc) => {
+        const documentData = doc.docs.map((d) => d.data());
+
+        // Convert the document data to a JSON string
+        const jsonString = JSON.stringify(documentData);
+
+        // Write the JSON string to a file
+        writeFileSync('/tmp/document.json', jsonString);
+
+        // Upload the file to Firebase Cloud Storage
+        try{
+            // Get bucket
+            var bucket = storage().bucket();
```

Comment:
Don't think we need to create temp files anywhere. Just write straight to the storage bucket.

![image](https://user-images.githubusercontent.com/2824906/216191054-260df840-09aa-4133-9530-93ce2f235b16.png)